homework-4/script.js

```diff
@@ -0,0 +1,247 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+  const baseUrl = 'http://localhost:3000';
+  const iconCloseBtn = document.querySelector('.icon-close-btn');
+  const headCheckbox = document.querySelector('.head-checkbox');
+  const tableBody = document.querySelector('.table-body');
+  const companyName = document.querySelector('.company-name');
+  const contactName = document.querySelector('.contact-name');
+  const address = document.querySelector('.address');
+  const city = document.querySelector('.city');
+  const country = document.querySelector('.country');
+  const usersBtn = document.querySelector('.add-users');
+  const updateUsers = document.querySelector('.update-users');
+  const addCumstomerBtn = document.querySelector('.add-customer-btn');
+  const close = document.querySelector('.close');
+  const addName = document.querySelector('.add-name');
+  const addContact = document.querySelector('.add-contact');
+  const addAddress = document.querySelector('.add-address');
+  const addCity = document.querySelector('.add-city');
+  const addCountry = document.querySelector('.add-country');
+  const modalAdd = document.querySelector('.modal-add');
+  const modal = document.querySelector('.modal-edit-object');
+  const modalClose = document.querySelector('.modal-close');
+
+  fetch(`${baseUrl}/posts`)
+  .then(data => data.json())
+  .then(res => 
+    res.map(function(element) {
+      createNewTaskElement(element);
+          
+      const deleteBtn = document.querySelectorAll('.delete');
+      for (let i = 0, len = deleteBtn.length; i < len; i++) {
+        deleteBtn[i].addEventListener('click', (ele) => {
+          let targetElement = ele.currentTarget;   
+          let id = targetElement.getAttribute('data-id');
```

Comment:
`const id = ele.currentTarget.getAttribute('data-id');`
no need to declare additional variable.

also please read about using data-attribetes: https://developer.mozilla.org/ru/docs/Learn/HTML/Howto/Use_data_attributes